README.md

```diff
@@ -2,6 +2,19 @@
 
 GOG Galaxy 2.0 Community integration for Steam.
 
+## Installation
+
+*Latest release should be available directly from Galaxy*
+
+#### For custom build
+- close Galaxy
+- remove previously installed plugin folder (if any), usually<br>
+`%localappdata%\GOG.com\Galaxy\plugins\installed\steam_ca27391f-2675-49b1-92c0-896d43afa4f8`
+- create a folder of any name at the following path and copy the custom build to it:<br>
+`C:\Users\<User>\AppData\Local\GOG.com\Galaxy\plugins\installed\`
```

Comment:
```suggestion
`%localappdata%\GOG.com\Galaxy\plugins\installed\`
```

for consistency with the line above, but you can change the other way I you like. I have no preference, so use what looks more readable/convenient to you.